app/assets/javascripts/darkswarm/controllers/products_controller.js.coffee

```diff
@@ -76,12 +76,15 @@ Darkswarm.controller "ProductsCtrl", ($scope, $filter, $rootScope, Products, Ord
   $scope.appliedTaxonsList = ->
     $scope.activeTaxons.map( (taxon_id) ->
       Taxons.taxons_by_id[taxon_id].name
-    ).join(" #{t('products_or')} ") if $scope.activeTaxons?
+    ).join($scope.filtersJoinWord()) if $scope.activeTaxons?
 
   $scope.appliedPropertiesList = ->
     $scope.activeProperties.map( (property_id) ->
       Properties.properties_by_id[property_id].name
-    ).join(" #{t('products_or')} ") if $scope.activeProperties?
+    ).join($scope.filtersJoinWord()) if $scope.activeProperties?
+
+  $scope.filtersJoinWord = ->
+    $sce.trustAsHtml(" <span>#{t('products_or')}</span> ")
```

Comment:
can you explain this magic a little better in the commit message?

Reply:
I'll update the message :+1: 

Basically we need to build a really dynamic structure on the page like "Results for: `<search-term>` in `<filter1>` or `<filter2>` or `<filter3>`" and the styling on each of the words needs to be a bit different, specifically the filter terms need to be differently styled to the join words like "in" and "or". We have to generate those bits on the Angular side, but they also have to be wrapped in HTML in a specific way. It was a bit tricky to achieve. 

`$sce.trustAsHtml()` allows us to add some trusted HTML in the strings, in this case the content doesn't involve any user-defined data, so it should be fine.

Reply:
Lengthy explanation added :+1: :smile:

The diff on the individual commit is relatively clear, when comparing it to what the old line was doing previously.

Reply:
awesome, thanks for explanning :+1: